cmd/cve/generate.go

```diff
@@ -44,11 +45,22 @@ func main() {
 		panic(err)
 	}
 
+	// Download the last released NVD feed
+	logger.Log("msg", "Downloading latest release")
+	if err := downloadLatestRelease(*dbDir, *debug, logger); err != nil {
+		panic(err)
```

Comment:
Should we panic?

Double checking that this optimization does retries in case of network errors in the action (NVD 2.0 is flaky).

Reply:
🤔 true we don't need to panic here.  If it fails to download the latest release from GitHub it will force a full sync from NVD

Reply:
OK, maybe log a warning (or we'll know because it took longer than expected :)

Reply:
I removed the panic and added retries on error, ending with a warning log